ADR/00xx-related-pipelines.md

```diff
@@ -0,0 +1,103 @@
+  * In this case, the bundle image must be rebuilt for the operand image update to be really be evaluated when deployed by OLM.
+  * This is a one-to-many dependency. One operator bundle depends on many operand images.
+* A user team just has **two components that depend on each other in a functional way**. When they propose an update to one component, they want to submit a corresponding change to the second component, and have those tested together before merging both.
+  * In this case, on any given day, component B’s PRs may or may not have dependencies on PRs from component A. Not all changes are linked.
+  * This is a many-to-many dependency. The user may have lots of components that depend on lots of other components.
+
+In theory **any combination of the cases above** could be present in an app. They could have one common parent image, many operand images that depend on that, and one or more operator bundles which depend on those.
+
+Today, users work around how complicated it is to manage digests themselves by instead using *floating tags*, which have the benefit of being easy to use - no need to update - but have the problem of being unclear. It's not exactly clear what you're building against if you refer to it by tag. Potentially insecure. We want to make it easy for users to do better.
+
+## Decision
+
+In order to support linking of multiple components, any component dependency will be encoded as part of the Tekton definition to be run for **pull request** event types.
+
+### User flow
+
+Users will be able to specify dependencies between components by modifying the pull request Tekton definition or by adding configuration to the UI which will result in AppStudio encoding the configuration in the Tekton definition. This definition will allow the specification of whether dependent components should be run in parallel or serially when combined.
```

Comment:
"This definition will allow the specification of whether dependent components should be run in parallel or serially when combined."

why would it be needed?

if A & B is dependent on C, why would you run A & B ever serially?

the only use case I can think of is to safe resources, but at the price of slowing this rebuild process